jobs.groovy

```diff
@@ -600,14 +678,17 @@ def copy_artifacts_from(context, full_job_name, short_job_name, archive_artifact
     }
 }
 
+PULL_REQUEST_JOB_TYPES = ['run_trial', 'run_trial_for_storage_driver',
+                          'run_acceptance', 'run_sphinx', 'omnibus',
+                          'run_lint', 'run_client']
```

Comment:
:eyes: No action required. I guess that we're getting to the stage where "job type" should be an object with properties like "appear in pull requests".